device_plugins/deploy_qat.md

```diff
@@ -1,4 +1,16 @@
 # Create Intel QAT Device Plugin CR
+Intel® QuickAssist Technology (Intel® QAT) provides cryptographic and compression acceleration capabilities used to improve performance and efficiency across the data center.
+For more details see https://intel.github.io/quickassist
+
+## Prerequisite
+Before deploying the QAT plugin, make sure iommu MCO is deployed and iommu is enabled.
+By default, half of the virtual functions are used for compression and the other half is used for cryptography. This default behaviour can be customised via a config map which configures the hardware using sysfs. The config values could be `asym`, `sym` and `dc`, for more details see https://github.com/torvalds/linux/blob/master/Documentation/ABI/testing/sysfs-driver-qat
+
+To create a configmap, run the following command:
+```
+kubectl create configmap --namespace=inteldeviceplugins-system --from-literal "qat.conf=ServicesEnabled=<option>" qat-config
```

Comment:
one change- kubectl to oc and namespace might be openshift-operators/namespace where plugin is created. Thoughts?@mregmi

Reply:
that was my mistake. fixed.